apps/editors/views.py

```diff
@@ -342,10 +351,18 @@ def save_motd(request):
     if form.is_valid():
         set_config('editors_review_motd', form.cleaned_data['motd'])
         return redirect(reverse('editors.motd'))
-    data = context(form=form)
+    data = context(request, form=form)
     return render(request, 'editors/motd.html', data)
 
 
+def is_admin_reviewer(request):
+    return acl.action_allowed(request, 'ReviewerAdminTools', 'View')
+
+
+def exclude_admin_only_addons(queryset):
+    return queryset.filter(admin_review=False, has_info_request=False)
```

Comment:
My non-admin reviewer appears to be able to review versions with `has_info_request=True` so I'm not sure if we should filter on this? Maybe not until the need info flagged is cleared more aggressively.